src/hooks.server.ts

```diff
@@ -0,0 +1,7 @@
+import type { RequestEvent } from '@sveltejs/kit';
```

Comment:
svelte update has a difference between server / client in some way. so this file changed to be this.